x-pack/plugins/apm/public/utils/formatters.ts

```diff
@@ -25,21 +26,42 @@ export function asMillis(value: number, withUnit = true) {
   return `${formatted}${withUnit ? ' ms' : ''}`;
 }
 
+export function asMicros(value: number, withUnit = true) {
```

Comment:
These bare boolean args are not my favorite (`asMicros(5000000, false)` :-1:) but I see we're already using it with `asSeconds` so the consistency is fine but maybe we can improve that later, or maybe TS makes this not as bad? Fine for now :)

Reply:
I could change it to something like:

```ts

interface FormatterArgs {
  value: number;
  unit: boolean;
  defaultValue: string;
}

export function asMillis({
  value,
  unit = true,
  defaultValue = 'N/A'
}: FormatterArgs) {
  if (value == null) {
    return defaultValue;
  }

  const formatted = asInteger(value / 1000);
  return `${formatted}${unit ? ' ms' : ''}`;
}
```

Then I can also delete `asMillisWithDefault`.